skbio/io/format/_base.py

```diff
@@ -227,13 +229,18 @@ def _line_generator(fh, skip_blanks=False, strip=True):
 def _too_many_blanks(fh, max_blanks):
     count = 0
     too_many = False
+    consumed = []
     for line in _line_generator(fh, skip_blanks=False):
+        consumed.append(line)
         if line:
             break
         else:
             count += 1
             if count > max_blanks:
                 too_many = True
                 break
-    fh.seek(0)
-    return too_many
+    try:
+        fh.seek(0)
+    except io.UnsupportedOperation:
+        pass
```

Comment:
If the `seek` doesn't matter, why do it?

Reply:
From below, it looks like a call to `.seekable` may work rather than a try/except?